nautobot/extras/urls.py

```diff
@@ -530,4 +531,15 @@
         name="webhook_changelog",
         kwargs={"model": Webhook},
     ),
+    # Notes
+    path("notes/add/", views.NotesEditView.as_view(), name="notes_add"),
+    path("notes/<slug:slug>/", views.NotesView.as_view(), name="notes"),
+    path(
+        "notes/<slug:slug>/changelog/",
+        views.ObjectChangeLogView.as_view(),
+        name="notes_changelog",
+        kwargs={"model": Notes},
+    ),
+    path("notes/<slug:slug>/edit/", views.NotesEditView.as_view(), name="notes_edit"),
+    path("notes/<slug:slug>/delete/", views.NotesDeleteView.as_view(), name="notes_delete"),
```

Comment:
I do think it'd be desirable to have a changelog-like URL pattern as well where a user can view `/app/model/<id>/notes/` to get a list of all notes specific to that particular object. That's a lot of places in `urls.py` to touch at present but something like #1902 should make that a lot simpler in the long run.

Reply:
I don't disagree, but the idea I was modeling is Dynamic Groups, with the tabs not being their own view, but instead is just a table on a tab. You can still go directly to the URL ie. `/app/model/<id>/#notes`.

I also encountered issues when having a separate URL, you could not go from /notes to /changelog, the tab would disappear (due to losing data when changing endpoints). The second was just reducing the amount of changes that a Plugin or maintainer would need to change in order to take advantage of the notes. Perhaps there is a better way to append the URL.